app/src/main/java/fi/bitrite/android/ws/ui/MapFragment.java

```diff
@@ -128,7 +131,9 @@
 
     private boolean mHasEnabledLocationProviders;
     private final BehaviorSubject<Location> mLastDeviceLocation = BehaviorSubject.create();
+
     private MyLocationNewOverlay mDeviceLocationOverlay;
+    private int mapZoomMinLoad;
```

Comment:
I’m not a fan of the hungarian notation with the preceding `m` for member variables, but since the rest of the app (and this class as well) uses it, this should be changed.